src/resources/styles.ts

```diff
@@ -71,6 +71,10 @@ export const haStyleDialog = css`
     --mdc-theme-primary: var(--google-red-500);
   }
 
+  .paper-dialog-buttons .warning.left-align {
+    margin-right: auto;
+  }
```

Comment:
Move this to `hui-edit-views` `static get styles()`, there is a `.delete` class there that was used for the button, should clean that up and replace with the new styles.